lib/Ama.pm

```diff
@@ -113,6 +120,10 @@ $self->plugin("OAuth2Accounts" => {
 
   $api->post('/:entry_type/vote/:entry_id/:vote', [vote => [qw(up down)]])->to('votes#cast')->name('cast_vote');
   $api->delete('/:entry_type/vote/:entry_id')->to('votes#uncast')->name('uncast_vote');
+
+  
+  $api->post('/feedback_comment')->to('emailfeedback#submit')->name('submit_emailfeedback');
```

Comment:
Interesting...  Why is there a feedback route on L105 and L125?  The one on L105 seems wrong to me.